src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -132,6 +136,10 @@ def initData(self,
         with h5py.File(self._file_name, "r") as h5file:
             nxprocess = h5file[self._nxprocess_path]
             map_data = get_dataset(nxprocess, "result/intensity")[()].sum(axis=-1)
+            try:
+                self._map_ptr = get_dataset(nxprocess, "result/map_ptr")[()]
+            except Exception:
```

Comment:
`Exception` can probably be more specific ? `KeyError` ?

Reply:
Probably, the original code is sometimes checking against both of them. I will migrate them all to KeyError if possible